webpack.config.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+const path = require('path')
+const HtmlWebPackPlugin = require('html-webpack-plugin')
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin')
+const PACKAGE = require('./package.json')
+const version = PACKAGE.webpack_bundle_version
+const CleanWebpackPlugin = require('clean-webpack-plugin')
+const project_config = require('./config.js')
+
+const fs = require('fs')
```

Comment:
`fs` is only being used in `generateHtmlPlugins`, we should define the variable there to prevent unnecessary scope creep